source/content/plugins-known-issues.md

```diff
@@ -338,6 +338,37 @@ ___
 
 ___
 
+## [GDPR Cookie Consent](https://wordpress.org/plugins/cookie-law-info/)
+
+<ReviewDate date="2020-02-20" />
```

Comment:
Might be a known issue already, but review date renders incorrectly - this says 20th, but rendered output produces 19th (not isolated to this instance, it appears widespread)

Reply:
Yea, that's fixed in a pending commit: ce86eeda7a43c11c25c281c7fd76391c7c80238c